docs/batfish.config.js

```diff
@@ -40,7 +40,12 @@ module.exports = () => {
     },
     dataSelectors: {
       navigation: (data) => buildNavigation(siteBasePath, data),
-      topics: (data) => buildTopics(data, appendTopics),
+      topics: (data) =>
+        buildItems({ data, append: appendTopics, itemType: 'topics' }),
+      level: (data) => buildItems({ data, itemType: 'level' }),
+      language: (data) => buildItems({ data, itemType: 'language' }),
+      products: (data) => buildItems({ data, itemType: 'products' }),
+      videos: (data) => buildItems({ data, itemType: 'videos' }),
```

Comment:
I was not able to fully investigate how we might want to build `append[Item]`s for each of the other "filterables."